vms/vm-under-test/scripts/customize-vm

```diff
@@ -27,3 +27,6 @@ dnf --enablerepo=rt install -y tuned-profiles-realtime
 dnf --enablerepo=nfv install -y tuned-profiles-nfv-guest
 
 grubby --args="iommu=pt intel_iommu=on default_hugepagesz=1GB hugepagesz=1G hugepages=1" --update-kernel=ALL
+
+swapoff -a
```

Comment:
Isn't this command redundant if the swap is removed from `/etc/fstab`?

Reply:
no. I tried